src/site/content/en/media/accessibility/index.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
@rkochman Kayce asked me make sure we covered everything in this article: https://www.html5rocks.com/en/tutorials/video/basics/

It had some accessibility material that I didn't have. I rewrote the section to include it. Please take a look.

Reply:
LGTM. We're going to include the Javascript content in phase 2, correct?